style.css

```diff
@@ -358,3 +388,432 @@
+  .body-area {
+    background-image: url(media/Header-llustration-desktopHeader.svg);
+    background-size: contain;
+    background-repeat: no-repeat;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-self: center;
+  }
+
+  .header-cover {
+    display: none;
+  }
+
+  .big-back-ground {
+    width: 100%;
+  }
+
+  .nav-bar-ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    list-style: none;
+    padding: 0;
+    gap: 8px;
+    margin-top: 16px;
+    margin-bottom: 16px;
+  }
+
+  .nav-bar-ul a {
+    text-decoration: none;
+  }
+
+  .nav-bar-ul li {
+    padding: 10px 12px;
+    width: 87px;
+    height: 40px;
+    font-size: 15px;
+    font-weight: 600;
+    line-height: 20px;
+    color: #344563;
+    text-align: center;
+  }
+
+  .main-container {
+    display: block;
+    justify-content: center;
+    align-items: center;
+    width: auto;
+    margin: 0% 10%;
+  }
+
+  .main-text {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 100px 0 200px 0;
+    gap: 24px;
+    margin: -5% 10%;
+  }
+
+  .main-text-blue {
+    font-family: "Inter", Arial, Helvetica, sans-serif;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 60px;
+    color: #172b4d;
+    text-align: center;
+    letter-spacing: 0.37px;
+  }
+
+  .main-text-green {
+    font-family: "Inter", Arial, Helvetica, sans-serif;
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
+    color: #36b37e;
+    text-align: center;
+    letter-spacing: 0.37px;
+    margin-top: -30px;
+  }
+
+  .intro-text {
+    font-size: 20px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: 400;
+    line-height: 32px;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  .social-media {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: center;
+    list-style: none;
+    width: 200px;
+    height: 40px;
+  }
+
+  .social-media li {
+    padding: 8px;
+  }
+
+  .work-container {
+    width: 100%;
+    margin-top: -100px;
+  }
+
+  .work-history {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    gap: 10px;
+    width: 100%;
+    height: auto;
+    left: 135px;
+    top: 1059px;
+  }
+
+  .my-work-text {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    margin: 5% 20%;
+  }
+
+  .my-work-text-heading {
+    font-weight: 700;
+    font-size: 20px;
+    color: #172b4d;
+    text-align: center;
+    gap: 24px;
+  }
+
+  .hr-line {
+    display: none;
+  }
+
+  .project-preview img {
+    width: 100%;
+    height: auto;
+    border-radius: 8px;
+  }
+
+  .project {
+    display: flex;
+    flex-direction: column;
+    border: 1px solid #d0d9d4;
+    border-radius: 5px;
+  }
+
+  .project-info {
+    display: flex;
+    flex-direction: column;
+    border-radius: 8px;
+    text-align: center;
+    justify-content: center;
+  }
```

Comment:
- [ ] Kindly centralize all the elements in the card and add some padding/margin to the button so it does not stay on the edge of the project card.
 - Your design:
 - 
<img width="387" alt="image" src="https://user-images.githubusercontent.com/61919617/228855420-f456ec5f-b460-4991-9d44-0019b0974aa8.png">

 - Figma template:
 - 
<img width="322" alt="image" src="https://user-images.githubusercontent.com/61919617/228855520-9e88fa71-fde3-4db1-ba7c-b91448db3f86.png">